packages/one-app-bundler/__tests__/bin/webpack-bundle-module.spec.js

```diff
@@ -53,27 +60,36 @@ describe('bundle-module', () => {
     expect(localeBundler).toHaveBeenCalledWith(false);
   });
 
-  it('should bundle the module for the server', () => {
+  it('should bundle the module for the server', async () => {
     process.argv = [];
     require('../../bin/webpack-bundle-module');
+
+    await sleep();
```

Comment:
Why are we waiting on time here? can we not directly wait on the bundling to be done?

Reply:
gonna remove this code in this file since it isn't needed anymore but I also have it in `bundle-module.spec.js` because the `bundle-module` file calls an async function that needs to be resolved before I can run the assertions. It's a bit hacky but not sure how to wait until the promise is resolved when the file is required

Reply:
right, you would almost need to await on the require itself, which doesn't do that. Thanks for the context.

Reply:
correct, i could export the fn and call it with await but I feel like i won't be actually testing how this works